assets/js/app/Data.js

```diff
@@ -9,7 +9,13 @@ LocusZoom.DataSources = function() {
     this.sources = {};
 };
 
+LocusZoom.KnownDataSources = [];
```

Comment:
Each locuszoom instance can/should have it's own DataSources collection. The KnownDataSources are meant to be universal. It's the list of recipes while DataSources is a realization of a particular meal (where each dish is given some unique name). I don't see a logical way which they can be combined. Maybe I just need better names or some other level of abstraction.